modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm

```diff
@@ -0,0 +1,237 @@
+
+/datum/armament_entry/company_import/deforest/equipment/advanced_retractor
+	item_type = /obj/item/retractor/advanced
+	lower_cost = CARGO_CRATE_VALUE * 2
+	upper_cost = CARGO_CRATE_VALUE * 5
+	interest_addition = COMPANY_INTEREST_GAIN_AVERAGE
+	interest_required = HIGH_INTEREST
+
+/datum/armament_entry/company_import/deforest/equipment/advanced_cautery
+	item_type = /obj/item/cautery/advanced
+	lower_cost = CARGO_CRATE_VALUE * 2
+	upper_cost = CARGO_CRATE_VALUE * 5
+	interest_addition = COMPANY_INTEREST_GAIN_AVERAGE
+	interest_required = HIGH_INTEREST
+
+/datum/armament_entry/company_import/deforest/equipment/afad
+	item_type = /obj/item/gun/medbeam/afad
+	lower_cost = CARGO_CRATE_VALUE * 5
+	upper_cost = CARGO_CRATE_VALUE * 10
+	interest_addition = COMPANY_INTEREST_GAIN_BIG
+	interest_required = HIGH_INTEREST
+
+// Various advanced cybernetic organs, organ replacements for the rich
+
+/datum/armament_entry/company_import/deforest/cyber_organs
+	subcategory = "Premium Cybernetic Organs"
+
+/datum/armament_entry/company_import/deforest/cyber_organs/eyes
+	name = "shielded cybernetic eyes"
+	item_type = /obj/item/storage/organbox/advanced_cyber_eyes
+	lower_cost = CARGO_CRATE_VALUE * 2
+	upper_cost = CARGO_CRATE_VALUE * 5
+	interest_addition = COMPANY_INTEREST_GAIN_AVERAGE
+	interest_required = PASSED_INTEREST
+
+/datum/armament_entry/company_import/deforest/cyber_organs/ears
+	name = "upgraded cybernetic ears"
+	item_type = /obj/item/storage/organbox/advanced_cyber_ears
+	lower_cost = CARGO_CRATE_VALUE * 2
+	upper_cost = CARGO_CRATE_VALUE * 5
+	interest_addition = COMPANY_INTEREST_GAIN_AVERAGE
+	interest_required = PASSED_INTEREST
+
+/datum/armament_entry/company_import/deforest/cyber_organs/heart
+	name = "upgraded cybernetic heart"
+	item_type = /obj/item/storage/organbox/advanced_cyber_heart
+	lower_cost = CARGO_CRATE_VALUE * 2
+	upper_cost = CARGO_CRATE_VALUE * 5
+	interest_addition = COMPANY_INTEREST_GAIN_AVERAGE
+	interest_required = PASSED_INTEREST
+
+/datum/armament_entry/company_import/deforest/cyber_organs/liver
+	name = "upgraded cybernetic liver"
+	item_type = /obj/item/storage/organbox/advanced_cyber_liver
+	lower_cost = CARGO_CRATE_VALUE * 2
+	upper_cost = CARGO_CRATE_VALUE * 5
+	interest_addition = COMPANY_INTEREST_GAIN_AVERAGE
+	interest_required = PASSED_INTEREST
+
+/datum/armament_entry/company_import/deforest/cyber_organs/lungs
+	name = "upgraded cybernetic lungs"
+	item_type = /obj/item/storage/organbox/advanced_cyber_lungs
+	lower_cost = CARGO_CRATE_VALUE * 2
+	upper_cost = CARGO_CRATE_VALUE * 5
+	interest_addition = COMPANY_INTEREST_GAIN_AVERAGE
+	interest_required = PASSED_INTEREST
+
+/datum/armament_entry/company_import/deforest/cyber_organs/stomach
+	name = "upgraded cybernetic stomach"
+	item_type = /obj/item/storage/organbox/advanced_cyber_stomach
+	lower_cost = CARGO_CRATE_VALUE * 2
+	upper_cost = CARGO_CRATE_VALUE * 5
+	interest_addition = COMPANY_INTEREST_GAIN_AVERAGE
+	interest_required = PASSED_INTEREST
```

Comment:
do me a favor while you're here and please swap the retarded-ass FAILED_INTEREST, PASSED_INTEREST, HIGH_INTEREST defines for more understandable names pls